### PR TITLE
Add more words about standard vs normal

### PR DESCRIPTION
**Pull request by rain-on**
Add more information to aid people's understanding around normal vs standard changes, and extend troubleshooting - incrementally.

### CODE REVIEW

**Review by harrisonmeister — APPROVED** [review]

File: docs/approvals/service-now/index.md [hunk 1]
```diff
@@ -117,9 +120,17 @@ To enable a project to enforce a requirement for an approved CR:
 
 ![ServiceNow Integration Project settings](images/servicenow-project-settings.png "width=500")
 
-### Standard Change Templates
+### Standard vs Normal Changes
+
+By default deployments resulting in CR creation will produce a `Normal` change (i.e. one
```
Comment (long-sean) [inline docs/approvals/service-now/index.md:125]:
```suggestion
By default, deployments resulting in CR creation will produce a `Normal` change (i.e. one 
```

File: docs/approvals/service-now/index.md [hunk 2]
```diff
@@ -117,9 +120,17 @@ To enable a project to enforce a requirement for an approved CR:
...
+
+By default deployments resulting in CR creation will produce a `Normal` change (i.e. one 
+requiring explicit approval).
 
-By default, the deployments resulting in a CR creation will produce a `Normal` change. Setting the **Change Template Name** setting under **Project Settings** to the name of a valid, approved **Change Template** will instead create a `Standard` change based upon the change template.
+Setting the **Standard Change Template Name** setting under **Project Settings** to the name of an 
+active, approved **Standard Change Template** (as found in the Standard Change Catalog) will instead 
+result in deployments of the project creating a `Standard` (i.e. low-risk, pre-approved) change. 
+
+For more information on `Normal` vs `Standard` changes, see ServiceNow [Change Types](https://docs.servicenow.com/en-US/bundle/tokyo-it-service-management/page/product/change-management/concept/change-types.html)
```
Comment (long-sean) [inline docs/approvals/service-now/index.md:132]: It is unfortunate that we get tied to a version via the URL. Wonder if the slightly less convenient generic link is acceptable, e.g. 

```md
For more information on `Normal` vs `Standard` changes, search for 'Change Types' on ServiceNow's [documentation site](https://docs.servicenow.com/) using your specific ServiceNow version.
```

**Review by long-sean — COMMENTED** [review]
Some suggestions, changes at your discretion

File: docs/approvals/service-now/index.md (hunk 2 above)
Comment (rain-on) [reply docs/approvals/service-now/index.md:132]: I think it's either we remove this _entirely_, or target latest version.

**Pull request merged**